quickstart/configure-mayastor.md

```diff
@@ -1,45 +1,45 @@
 # Configure Mayastor
 
-## Create Mayastor Pool\(s\)
+## Create DiskPool\(s\)
 
 
-### What is a Mayastor Pool?
+### What is a DiskPool?
 
-When a Mayastor node allocates storage capacity for a replica of a Persistent Volume it does so from a Mayastor Pool. Each Mayastor node may create and manage zero, one, or more such pools. The ownership of a pool by a node is exclusive. A pool can manage only one block device, which constitutes the entire data persistence layer for that pool and thus defines its maximum storage capacity.
+When a node allocates storage capacity for a replica of a persistent volume(PV) it does so from a DiskPool. Each node may create and manage zero, one, or more such pools. The ownership of a pool by a node is exclusive. A pool can manage only one block device, which constitutes the entire data persistence layer for that pool and thus defines its maximum storage capacity.
 
 {% hint style="info" %}
-Note that in the current version of Mayastor pools support only thick provisioning.
+The current version of DiskPools only supports thick provisioning.
 {% endhint %}
 
-A pool is defined declaratively, through the creation of a corresponding `MayastorPool` (MSP) custom resource on the cluster. The MSP must be created in the same namespace as Mayastor has been deployed.  User configurable parameters of this resource type include a unique name for the pool, the node name on which it will be hosted and a reference to a disk device which is accessible from that node. The pool definition requires the reference to its member block device to adhere to a discrete range of schemas, each associated with a specific access mechanism/transport/ or device type.
+A pool is defined declaratively, through the creation of a corresponding `DiskPool` custom resource on the cluster. The DiskPool must be created in the same namespace where Mayastor has been deployed.  User configurable parameters of this resource type include a unique name for the pool, the node name on which it will be hosted and a reference to a disk device which is accessible from that node. The pool definition requires the reference to its member block device to adhere to a discrete range of schemas, each associated with a specific access mechanism/transport/ or device type.
 
-#### Permissible Schemes for the MSP field `disks`
+#### Permissible Schemes for `spec.disks` under DiskPool CRD
 
 | Type | Format | Example |
 | :--- | :--- | :--- |
-| Attached Disk Device \(AIO\) | Device File | /dev/sdx |
-| Async. Disk I/O \(AIO\) | Device File | aio:///dev/sdx |
+| Asynchronous Disk\(AIO\) | Device File | /dev/sdx |
+| Asynchronous Disk I/O \(AIO\) | Device File | aio:///dev/sdx |
```

Comment:
Lines 20 and 21 seem the same.

Reply:
There is just a slight change in the example. The second one has "aio".